imagersite/templates/imagersite/home.html

```diff
@@ -0,0 +1,17 @@
+{% extends "imagersite/base.html" %}
+{% block body %}
+<div id="home" class="meta">
+  <h2>Welcome to <strong>Image Library</strong></h2>
+  <h4>Share your photos with the world</h4>
+  <h4>Create an account or log in below</h4>
+  <img class="tree" src="{{ cover_path }}">
+  <div>
+    <a href="{% url 'auth_login' %}">
+      <input type="button" value="Login" class="btn btn-primary"></input>
```

Comment:
This and the input field below shouldn't be taking closing `</input>` tags. Inputs are self-closing tags.